### PR TITLE
[Merged by Bors] - feat: add Algebra.coe_trace_int

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
From flt-regular.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by alreadydone** [comment]
maintainer merge

File: Mathlib/NumberTheory/NumberField/Norm.lean [hunk 1]
```diff
@@ -26,13 +26,17 @@ open scoped NumberField BigOperators
...
 
-theorem Algebra.coe_norm_int {K : Type*} [Field K] [NumberField K] (x : 𝓞 K) :
-    Algebra.norm ℤ x = Algebra.norm ℚ (x : K) :=
+variable {K : Type*} [Field K] [NumberField K] (x : 𝓞 K)
+
+theorem Algebra.coe_norm_int : Algebra.norm ℤ x = Algebra.norm ℚ (x : K) :=
   (Algebra.norm_localization (R := ℤ) (Rₘ := ℚ) (S := 𝓞 K) (Sₘ := K) (nonZeroDivisors ℤ) x).symm
 
-end rat
+theorem Algebra.coe_trace_int : Algebra.trace ℤ _ x = Algebra.trace ℚ K x :=
```
Comment (alreadydone) [inline Mathlib/NumberTheory/NumberField/Norm.lean:36]: Maybe make one more argument implicit in `Algebra.trace` in a future PR?

Comment (eric-wieser) [inline Mathlib/NumberTheory/NumberField/Norm.lean:36]: Can you write the coercion explicitly with `↑` just to make the naming clear here?

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Comment by riccardobrasca** [comment]
bors merge

**Pull request closed without merging**